src/angular/datepicker/datepicker.md

```diff
@@ -12,7 +12,7 @@ The datepicker form field can have a toggle to show a calendar to pick dates fro
 and arrows which by clicking increase or decrease dates.
 
 ```html
-<sbb-datepicker toggle="false" arrows>
+<sbb-datepicker noToggle arrows>
```

Comment:
For some reason `noToggle` does not look nice to me. What do you think abount `no-toggle` or `notoggle`?